--- a/lib/module.js	2015-10-13 19:20:12.000000000 +0200
+++ b/lib/module.js	2015-10-20 11:20:56.000000000 +0200
@@ -478,7 +478,8 @@
     var homeDir = process.env.HOME;
   }

-  var paths = [path.resolve(process.execPath, '..', '..', 'lib', 'node')];
+  var paths = [path.resolve(process.execPath, '..', '..', 'lib', 'node'),
+    path.resolve(process.execPath, '..', '..', 'lib', 'node_modules')];

   if (homeDir) {
     paths.unshift(path.resolve(homeDir, '.node_libraries'));
