From 76b0521e59cb535bbdf36ccf328412ccdd933a20 Mon Sep 17 00:00:00 2001
From: Martino Facchin <m.facchin@arduino.cc>
Date: Thu, 12 Jan 2017 15:51:01 +0100
Subject: [PATCH 8/9] Send NVMCRTL write-back command after modifying USER ROW

---
 src/flash/nor/at91samd.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/flash/nor/at91samd.c b/src/flash/nor/at91samd.c
index f26096c0..0f72698b 100644
--- a/src/flash/nor/at91samd.c
+++ b/src/flash/nor/at91samd.c
@@ -517,6 +517,8 @@ static int samd_modify_user_row(struct target *target, uint32_t value,
 		goto out_user_row;
 	}
 
+	samd_issue_nvmctrl_command(target, SAMD_NVM_CMD_WAP);
+
 	/* Success */
 	res = ERROR_OK;
 
-- 
2.11.0

